src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import { fireEvent, getByRole, render, screen } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe('Search Box', () => {
+  const { asFragment } = render(<Search />)
```

Comment:
There is a warning in the console about required props being missing here. You should create a `validProps` object to house mocked versions of these to pass into the tests:
```
  const validProps = {
    setSearchResults: jest.fn(),
  };
```
Passing them in:
```
  it("search box renders correctly", () => {
    const { asFragment } = render(
      <Search setSearchResults={validProps.setSearchResults} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
```

Also, could some more tests be added here? You could assert that the function you are passing in is called when the search button is clicked for example